cloudify_types/cloudify_types/component/tests/test_deployment.py

```diff
@@ -163,7 +162,7 @@ def test_create_deployment_failed(self):
                     timeout=MOCK_TIMEOUT)
 
                 self.assertIn('No execution Found for component'
-                              ' \"test\" deployment', error.message)
+                              ' "test" deployment', error.message)
```

Comment:
if already changing this line, might as well make it `str(error)` as well instead of `error.message` (.message is deprecated since 2.6 and removed in 3)